src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -116,15 +132,33 @@ def load_command_table(self, _):
                                  client_factory=cf_acr_replications,
                                  table_transformer=replication_output_format)
 
+    with self.command_group('acr', acr_build_util) as g:
+        g.command('build', 'acr_build', client_factory=cf_acr_registries)
+
     with self.command_group('acr build-task', acr_build_task_util) as g:
         g.command('create', 'acr_build_task_create')
         g.command('show', 'acr_build_task_show')
         g.command('list', 'acr_build_task_list')
         g.command('delete', 'acr_build_task_delete')
-        g.command('list-builds', 'acr_build_task_list_builds', table_transformer=build_output_format,)
-        g.command('run', 'acr_build_task_run')
-        g.command('logs', 'acr_build_task_logs')
-
-    with self.command_group('acr build', acr_build_util) as g:
-        g.command('show-logs', 'acr_build_show_logs')
-        g.command('', 'acr_queue') # TODO: it should be moved to acr command group once we can integrate the full sdk.
+        g.generic_update_command('update',
```

Comment:
Not sure if we are still using the extension. If we are, then please update the command in init file of extension.

Reply:
Added.